435 (#464)
brazilian-utils/python: README_EN.md

None
```

## Boleto

### format_boleto

> This function takes information from a boleto
> and turns it into a string.

Reviewer: ```suggestion
This function takes information from a bank slip (in brazilian portuguese, boleto)
and turns it into a string.
```

**Args:**

> boleto (Boleto): A dictionary with boleto information

Reviewer: ```suggestion
boleto (Boleto): A dictionary with a bank slip information
```

**Returns:**

> str: A string with the formatted boleto reading code

Reviewer: ```suggestion
str: A string with the formatted bank slip reading code
```
